portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -23,52 +26,58 @@
 import com.google.gson.Gson;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Date;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that contains saves and gets comments from database. */
-@WebServlet("/data")
+/** Servlet that contains, saves and gets comments from database. 
+ *  Uses LoginServlet to determine if a user is logged in to
+ *  either show or hide comments.
+ */
+@WebServlet("/comments")
 public class DataServlet extends HttpServlet {
 
-    private List<String> comments = new ArrayList<>();
     private final Gson gson = new Gson();
+    UserService credentials = UserServiceFactory.getUserService();
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Whenever you declare fields in a class, mark them as `private final` unless you have a specific reason to do otherwise.